include/FastBVH/impl/BuildStrategy1.inl

```diff
@@ -58,6 +60,25 @@ class BuildStack final {
   const auto& operator[](std::size_t index) const noexcept { return entries[index]; }
 };
 
+//! \brief Node descriptor for the flattened tree
```

Comment:
I didn't have enough confidence that I could change your algorithm using the new `Node` structure, so I added this structure for backwards compatibility. Since, originally, there was a loop of code that copied "build nodes" to the nodes that get returned, if I figured I could translate the legacy nodes to the current ones there.